packages/fiori-freestyle/templates/worklist/add/webapp/controller/Worklist.controller.js

```diff
@@ -128,8 +126,13 @@ sap.ui.define([
 		 * @private
 		 */
 		_showObject : function (oItem) {
-			this.getRouter().navTo("object", {
-				objectId: oItem.getBindingContext().getProperty("<%=template.settings.entity.key%>")
+			var that = this;
+
+			oItem.getBindingContext().requestCanonicalPath().then(function (sObjectPath) {
+				that.getRouter().navTo("object", {
+					objectId_Old: oItem.getBindingContext().getProperty("ID"),
+					objectId : sObjectPath.slice("/Products".length) // /Products(3)->(3)
```

Comment:
This doesn't look right. I would expect a templating string instead of `Products`